File: Project7_LSH.py
import numpy as np
import pandas as pd


# 공통 데이터
data = {'a' : [1,2,3], 'b' : [4,5,6]}
c = pd.Series([1,2,pd.NA,4], index=(1,2,3,4))
print(c)
print(data)


df = pd.DataFrame(data, index=(1,2,3))
print(df)

#21. ndim (pandas)
print(df.ndim)

#22. size (pandas)
print(df.size)

#23. values (pandas)
print(df.values)

#24. head (pandas)
print(df.head())

#25. tail (pandas)
print(df.tail())

#26. shape (pandas)
print(df.shape)

#27. T(Transpose) (pandas)
print(df.T)

#28. describe (pandas)
print(df.describe())

#29. notnull (pandas)
print(c[c.notnull()])

#30. get_dummies()
pd.get_dummies(df['a'])

File: Project7_LSH.R
# 임성현

# 공통데이터
a = c(1,2,3)
b = c(4,5,6)
df = data.frame(a, b)
df
c = c(1,2,NA,4)

#21 ndim ; 차원만 따로 보여주는 기능을 못찾아서 dim으로 대체
dim(df)

#22 size
nrow(df)*ncol(df)

#23 values
array(c(a,b), dim = c(3,2,1))

#24 head
head(df)

#25 tail
tail(df)

#26 shape
dim(df)

#27 T
t(df)

#28 describe
summary(df)

#29 notnull
na.omit(c)

#30 get_dummies
table(df$a)





